.github/workflows/ci.yml

```diff
@@ -15,13 +15,14 @@ jobs:
       - uses: actions/checkout@master
       - uses: actions/setup-go@master
         with:
-          go-version: '1.21'
+          go-version: '1.23'
           cache: true
           cache-dependency-path: go.sum
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v4.0.0
```

Comment:
update `golangci/golangci-lint-action@v4.0.0` to `golangci/golangci-lint-action@v6`

Reply:
updated